src/Shell/Task/SeedTask.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * CakePHP(tm) : Rapid Development Framework (http://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (http://cakefoundation.org)
+ * @link          http://cakephp.org CakePHP(tm) Project
+ * @license       http://www.opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Migrations\Shell\Task;
+
+use Bake\Shell\Task\SimpleBakeTask;
+use Cake\Console\ConsoleOptionParser;
+use Cake\Core\Configure;
+use Cake\Core\Plugin;
+use Cake\Utility\Inflector;
+
+/**
+ * Task class for generating seed files.
+ */
+class SeedTask extends SimpleBakeTask
+{
+    /**
+     * path to Migration directory
+     *
+     * @var string
+     */
+    public $pathFragment = 'Seed/';
```

Comment:
Can't do that in a class attribute. Also, all other bake tasks use slashes, and php now correctly handles them afaik.